public/app/plugins/datasource/loki/responseUtils.ts

```diff
@@ -47,6 +49,25 @@ export function extractLabelKeysFromDataFrame(frame: DataFrame): string[] {
   return Object.keys(labelsArray[0]);
 }
 
+export function extractUnwrapLabelKeysFromDataFrame(frame: DataFrame): string[] {
```

Comment:
Do you think we could reuse this function also in Unwrap editor in query builder. So we have unified logic in both: https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/querybuilder/components/UnwrapParamEditor.tsx#L74

Reply:
Absolutely! Good idea.

Reply:
Done! Let me know what you think.

Reply:
Done.